### ISSUE TITLE
Add to METADATA

### ISSUE THREAD

**Issue by dlfivefifty** [mbauman/RaggedArrays.jl#9]
I'd like to use this package in ApproxFun.jl

**Comment by mbauman** [comment]
Sure.  I hadn't for a few reasons:
- I'm not currently using this for my own work.  It was an experiment to see if I could make it functional and strict (that is, it should error rather than give an unexpected result).  I think I've reached that point.
- ~~Missing #1 means that it allocates on eachindex and is pretty slow.  But the status quo is functional.~~ Now fixed.
- I'm still not convinced that it should be a subtype of AbstractArray, but I'd be curious to hear how you want to use it — does it need to be a subtype of AbstractArray in your use case?  Right now, I think the only abstract functionality that actually works and is useful is the display routines (and that's because it catches errors and marks out-of-ragged-length elements as `#undef`).  Some reductions might work over contiguous sections, but I'm not sure.  This really is predominantly a storage type that's not meant for direct computation.

Of course, we can always change points 2 and 3, even after registering it.

Is it "just working" as you'd expect in ApproxFun?

**Comment by mbauman** [comment]
Also: what versions do you need to support for ApproxFun?

**Comment by dlfivefifty** [comment]
Right now just 0.4, but eventually 0.5 too.

**Comment by mbauman** [comment]
After thinking about this some more, I'm not sure I'm willing to officially support this package right now… at least not without a few collaborators.  As it stands, it's just a storage container — there really aren't many AbstractArray algorithms in base that actually work.  There'd be a fairly substantial effort required to even get simple things like `sum` working.  If you just need a storage container, then maybe we can work out a compromise… but I'd like to hear about how it's working for you before registering it.

**Comment by dlfivefifty** [comment]
Fair enough.  I think I actually want my wrapper to act like a vector whose entries are vectors (a faster Vector{Vector{Float64}}, basically), so R[k][j] instead of R[j,k].  So I'm probably better off making my own ragged array data structure.

**Comment by mbauman** [comment]
Ah, interesting.  Yes, I highly recommend that approach; I think you'll have much greater success getting to something that "just works" with relatively minimal effort that way.
